pages/index.vue

```diff
@@ -1,4 +1,13 @@
 <template>
     <div>
+        <SearchPanel />
     </div>
 </template>
+
+<style>
+main{ 
+    background: linear-gradient(rgba(25, 25, 25, 0.7),
+                    rgba(25, 25, 25, 0.7)), url("../assets/background.jpg")no-repeat center center/cover;
```

Comment:
don't use style in vue page make it a different file.You can't give style main tag,you can create class and in that add style

Reply:
Corrected..